core/DataAccess/LogQueryBuilder.php

```diff
@@ -249,11 +249,27 @@ private function buildSelectQuery($select, $from, $where, $groupBy, $orderBy, $l
 				$orderBy";
         }
 
-        $limit = (int)$limit;
-        if ($limit >= 1) {
-            $sql .= "
-            LIMIT
-                $limit";
+        $sql = $this->appendLimitClauseToQuery($sql, $limit);
```

Comment:
There was a "bug". The PHPdoc already said it would support a Limit by clause but it only supported a limit number (int). It does now actually support `10, 5` for `LIMIT 5 OFFSET 10`